src/StateSmithTest/PlantUMLTests/ParsingTests.cs

```diff
@@ -473,6 +473,25 @@ @startuml blinky1_printf_sm
             """);
     }
 
+    /// <summary>
+    /// See https://github.com/StateSmith/StateSmith/issues/216
+    /// Ignore title line
+    /// </summary>
+    [Fact]
+    public void DiagramIgnoreTitle()
```

Comment:
Note that when I merge in the other PlantUML feature branch, we will also get a fix in the grammar. I moved the [`ignore` rule lower down in priority](https://github.com/StateSmith/StateSmith/commit/9ae94ae6da22b87d6a45301da1c8c357a05cef8a#diff-2bfa7d688bde81a86500d2557ed92c827310e9396528bf6510b99b38e1889813L156-R167).

Without that fix, we can't have states named 'title' like below:
```plantuml
@startuml system
[*] -> ACTIVE : POWER_BUTTON
ACTIVE -> IDLE : POWER_BUTTON
title -> IDLE /' this will get ignored right now '/
@enduml
```

Reply:
Just to be clear, you don't have to worry about the ignore rule in your PR.